test/common/network/udp_listener_impl_test.cc

```diff
@@ -314,6 +337,7 @@ TEST_P(UdpListenerImplTest, UdpListenerRecvMsgError) {
   // Inject mocked OsSysCalls implementation to mock a read failure.
   Api::MockOsSysCalls os_sys_calls;
   TestThreadsafeSingletonInjector<Api::OsSysCallsImpl> os_calls(&os_sys_calls);
+  EXPECT_CALL(os_sys_calls, supportsUdpGro());
```

Comment:
Do you  still need this?

Reply:
Since the MockOsSysCalls trumps the MockSupportUdpGro here. `EXPECT_CALL(os_sys_calls, supportsUdpGro());` is required, else the test fails with `uninteresting mock function ... supportsUdpGro`. Is there something else that I am missing here?

Reply:
makes sense. Thanks for explanation!